static/src/js/App.js

```diff
@@ -172,16 +179,29 @@ class App extends Component {
                         )
                       }
                     />
-                    <Route path={this.portalPaths('/projects')} component={Project} />
+                    <Route
+                      path={this.portalPaths('/projects')}
+                      render={
+                        () => (
+                          <AuthRequiredContainer>
+                            <Project />
+                          </AuthRequiredContainer>
+                        )
+                      }
+                    />
                     <Redirect exact from="/portal/:portalId/" to="/portal/:portalId/search" />
                     <Redirect exact from="/" to="/search" />
                     <Route
+                      // TODO not relevant to PR but, I want to understand why the Search component has to be loaded first
+                      // EVEN without the lazy loading
                       path={this.portalPaths('/search')}
                       render={
                         () => (
                           <>
                             <Search />
-                            <EdscMapContainer />
+                            <Suspense fallback={<Spinner type="dots" className="root__spinner spinner spinner--dots spinner--white spinner--small" />}>
```

Comment:
why is react-leaflet still getting pulled in?

Reply:
It looks like its getting pulled in by the `static/src/js/containers/SpatialSelectionContainer/SpatialSelectionContainer.js` which isn't under the MapContainer but, instead through the `Search` component